ta/pkcs11/src/handle.c

```diff
@@ -67,11 +67,16 @@ uint32_t handle_get(struct handle_db *db, void *ptr)
 	return n;
 }
 
+static bool db_and_handle_are_valid(struct handle_db *db, uint32_t handle)
```

Comment:
Even thou this is a static I would prefer to keep `handle_` prefix for method to make debugging life nicer.

Suggesting:
```c
static bool handle_is_valid(struct handle_db *db, uint32_t handle)
```